stable/spark/values.yaml

```diff
@@ -3,10 +3,24 @@
 # Declare variables to be passed into your templates.
 
 
+
+#ClusterRoleBinding parameters
+clusterRoleBinding:
+    name: default-pyspark
+    namespace: default
```

Comment:
ok, but if you specify the namespace here, you have to set the same namespace in every other resources (pod, svc etc)

Reply:
@ttedeschi did you solved this?

Reply:
Solved, I removed the namespace line

Reply:
Mmm, @ttedeschi I meant to put namespace everywhere else :) not to remove it, sorry.

Reply:
@dciangot done!